src/main/java/com/rk/rentacarapp/services/MyUserDetailsService.java

```diff
@@ -0,0 +1,30 @@
+package com.rk.rentacarapp.services;
+
+import com.rk.rentacarapp.repository.UserRepository;
+import org.springframework.security.core.userdetails.UserDetails;
+import org.springframework.security.core.userdetails.UserDetailsService;
+import org.springframework.security.core.userdetails.UsernameNotFoundException;
+import org.springframework.stereotype.Service;
+
+
+@Service
+public class MyUserDetailsService implements UserDetailsService {
+
+    private UserRepository userRepository;
+
+    public MyUserDetailsService(UserRepository userRepository) {
```

Comment:
Classes shouldn't be named with My...

This is an implementation of UserDetailService so with the naming convention from the rest of the project this should be UserDetailsServiceImpl